airmozilla/settings/base.py

```diff
@@ -224,6 +224,9 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 # Always use secure cookies
 COOKIES_SECURE = True
 
+#Default for related content
```

Comment:
hash characters for comments should have a space after them. See other comments.